BunqSdk/Http/ApiClient.cs

```diff
@@ -99,7 +99,7 @@ public ApiClient(ApiContext apiContext)
 
         private HttpClient CreateHttpClient()
         {
-            return new HttpClient(CreateHttpClientHandler())
+            return new HttpClient(new RetryHandler(CreateHttpClientHandler()))
```

Comment:
can the retry handler be included inside the `CreateHttpClientHandler` method ? 

Looks better.

Reply:
I did this based on the example, but this can be done I think.

Reply:
When I move the RetryHandler is it showing some errors.

Reply:
Kept it on the same place due to error when changing it.